### PR TITLE
[zh] Tweak kubelet credential provider

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md [hunk 1]
```diff
@@ -115,13 +117,14 @@ providers:
...
+    # 可以在域中使用通配符，但不能在端口或路径中使用。
+    # 支持通配符作为子域（例如 "*.k8s.io" 或 "k8s.*.io"）和顶级域（例如 "k8s.*"）。
+    # 还支持匹配部分子域，如 "app*.k8s.io"。
+    # 每个通配符只能匹配一个子域段，因此 "*.io" 不匹配 "*.k8s.io"。
     #
     # 当以下所有条件都为真时，镜像和 matchImage 之间存在匹配：
+    #
     # - 两者都包含相同数量的域部分并且每个部分都匹配。
     # - imageMatch 的 URL 路径必须是目标镜像 URL 路径的前缀。
     # - 如果 imageMatch 包含端口，则该端口也必须在图像中匹配。
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md:130]:
```suggestion
    # - 如果 imageMatch 包含端口，则该端口也必须在镜像中匹配。
```

**Review by Sea-n — COMMENTED** [review]
/approve

File: content/zh-cn/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md]: good catch.

**Comment by windsonsea** [comment]
We can consider adding `image registry 镜像库` to [zh-cn glossary](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/).

/lgtm

**Pull request merged**